omod/src/main/java/org/openmrs/module/htmlformentryui/fragment/controller/htmlform/EnterHtmlFormFragmentController.java

```diff
@@ -327,11 +346,15 @@ private void keepTimeComponentOfEncounterIfDateComponentHasNotChanged(Date previ
 	}
 	
 	private void setupModel(FragmentModel model, FormEntrySession fes, VisitDomainWrapper visitDomainWrapper,
-	        Boolean createVisit) {
+	        Boolean createVisit, String visitStartDatetime, String visitStopDatetime, UiUtils ui) {
 		
-		model.addAttribute("currentDate", (new DateMidnight()).toDate());
```

Comment:
Shouldn't we be using `DateMidnight` if we are not using time zones?

Reply:
We will do that on the client side with the `var extractDate = function (date) {`
https://github.com/openmrs/openmrs-module-htmlformentryui/pull/40/files/aa2f18528d1f768d25153aa2957fb84437c4af0c

Reply:
That midnight was only to force time to go 00:00:00